### ISSUE TITLE
Java build warning 

### ISSUE THREAD

**Issue by externl** [zeroc-ice/ice#2058]
```
/Users/joe/Developer/zeroc-ice/ice/java/src/Ice/src/main/java/module-info.java:12: warning: [module] module not found: com.zeroc.icebox
      com.zeroc.icebox;
```

The file has the contents:
```
module com.zeroc.ice {
  exports com.zeroc.Ice;
  exports com.zeroc.IceSSL;
  exports com.zeroc.Ice.Instrumentation;
  exports com.zeroc.Ice.IceMX;
  exports com.zeroc.IceInternal;
  exports com.zeroc.IceUtilInternal to
      com.zeroc.icebox;
}
```

Not obvious to me why it's structured like this. I don't think there's any icebox code in the ice module.

**Comment by externl** [comment]
I see, so what about the warning though?

**Comment by pepone** [comment]
My guess is that it is related to the Gradle setup, but I haven't looked too deeply into it.

**Comment by externl** [comment]
Sounds like this issue https://stackoverflow.com/questions/53670052/how-to-define-qualified-exports-to-unknown-modules

**Comment by InsertCreativityHere** [comment]
This was 'accidentally' fixed by #2770, which removed the `IceUtilInternal` package, fixing this warning.
